bigquery_etl/cli/stage.py

```diff
@@ -261,14 +261,14 @@ def _update_references(artifact_files, project_id, dataset_suffix, sql_dir):
                 re.compile(
                     rf"(?<![\._])`?{original_dataset}`?\.{name}(?![a-zA-Z0-9_])`?"
                 ),
-                f"`{deployed_project}.{deployed_dataset}.{name}`",
+                f"`{deployed_project}`.{deployed_dataset}.{name}",
             )
         )
         # replace fully qualified references (like "moz-fx-data-shared-prod.telemetry.main")
         replace_references.append(
             (
                 rf"(?<![a-zA-Z0-9_])`?{original_project}`?\.{original_dataset}`?\.{name}(?![a-zA-Z0-9_])`?",
```

Comment:
Seems to be missing optional backticks before `{original_dataset}` and `{name}`.
```suggestion
                rf"(?<![a-zA-Z0-9_])`?{original_project}`?\.`?{original_dataset}`?\.`?{name}(?![a-zA-Z0-9_])`?",
```